src/common/artifactQualityRevisionEntityAttributes.ts

```diff
@@ -0,0 +1,35 @@
+import { ArtifactRevision } from 'pnc-api-types-ts';
+
+import { artifactEntityAttributes } from 'common/artifactEntityAttributes';
+import { TEntityAttributes } from 'common/entityAttributes';
+
+interface IExtendedArtifactRevision extends ArtifactRevision {
+  'modificationUser.username': any;
+}
+
+export const artifactQualityRevisionEntityAttributes = {
+  id: {
+    id: 'id',
+    title: 'ID',
+  },
+  artifactQuality: {
+    id: 'artifactQuality',
+    title: 'Artifact Quality',
+    values: artifactEntityAttributes.artifactQuality.values,
+    filter: {
+      operator: '==',
+    },
+  },
+  modificationTime: {
+    id: 'modificationTime',
+    title: 'Modification Date',
+  },
```

Comment:
`Filtering by Quality` and `Sorting by Modification Time` feture seems not working from backend(http://openapi-newcastle-devel.psi.redhat.com/?urls.primaryName=orch-DEV-master#/Artifacts/getRevisions_2), do we have jira for this endpoint issue?

Reply:
Yes, I noticed the bug, see NCL-8060.